examples/undocumented/python/evaluation_cross_validation_mkl_weight_storage.py

```diff
@@ -65,18 +64,23 @@ def evaluation_cross_validation_mkl_weight_storage(traindat=traindat, label_trai
     cross_validation.set_autolock(False)
 
     # append cross vlaidation output classes
-    #cross_validation.add_cross_validation_output(CrossValidationPrintOutput())
-    mkl_storage=CrossValidationMKLStorage()
-    cross_validation.add_cross_validation_output(mkl_storage)
+    mkl_storage=ParameterObserverCV()
+    cross_validation.subscribe_to_parameters(mkl_storage)
     cross_validation.set_num_runs(3)
 
     # perform cross-validation
     result=cross_validation.evaluate()
 
     # print mkl weights
-    weights=mkl_storage.get_mkl_weights()
-    #print "mkl weights during cross--validation"
-    #print weights
+    weights = []
+    for o in mkl_storage.get_observations():
+        for fold in o.get_folds_results():
+            machine = MKLClassification.obtain_from_generic(fold.get_trained_machine())
```

Comment:
mmmh this cannot be translated into a meta example....
Maybe we can add an overly simplified one at least?
This one should go into the notebook on x-validation, we want to get rid of all language specific examples eventually